pkg/operator/ceph/object/controller.go

```diff
@@ -53,6 +53,7 @@ import (
 
 const (
 	controllerName = "ceph-object-controller"
+	DisableOBC     = "DISABLE_OBJECT_BUCKET_CLAIM"
```

Comment:
nit: We don't follow this pattern very consistently, but it's nice to name env vars. And the rook env vars mostly have the rook prefix.
```suggestion
	DisableOBCEnvVar = "ROOK_DISABLE_OBJECT_BUCKET_CLAIM"
```

Reply:
updated.